src/js/components/modals/InstallPackageModal.js

```diff
@@ -573,6 +581,7 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
 
     return (
       <Modal
+        ref="modal"
```

Comment:
Perhaps we should use callback refs instead, to avoid possible future tech debt? https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute